pkg/binary/envoy/adminport.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Tetrate
```

Comment:
I would like this to be in `pkg/binary/envoytest/util.go` since that is the only place this is used and these functions are only for tests.

Reply:
sg

Reply:
can't move this now as it needs access to the "envoy" package. I'll comment why.

We should probably make this a real feature in the near future, assuming it works for a while in tests!